IntegrationWithPharmacies/Controllers/ReportController.cs

```diff
@@ -32,7 +36,20 @@ public IActionResult Post(DateOfOrder date)
         {
             var sftpService = new SftpService(new NullLogger<SftpService>(), getConfig());
             var testFile = @"..\TextFile.txt";
-            System.IO.File.WriteAllText(testFile, getReportText(date));
+            StringBuilder stringBuilder2 = new StringBuilder();
+
+            List<RegistrationInPharmacy> registrations = RegistrationInPharmacyService.GetAll();
+            foreach (RegistrationInPharmacy registration in registrations)

```

Comment:
Write this in form:
"foreach (RegistrationInPharmacy registration in RegistrationInPharmacyService.GetAll())"